src/auto-evo/simulation/MichePopulation.cs

```diff
@@ -44,12 +37,41 @@ public static class PopulationSimulation
 
         while (parameters.StepsLeft > 0)
         {
-            RunSimulationStep(parameters, speciesToSimulate, patchesList, random, cache,
-                parameters.AutoEvoConfiguration, parameters.WorldSettings);
+            RunSimulationStep(parameters, speciesToSimulate, patchesList, random, cache);
             --parameters.StepsLeft;
         }
     }
 
+    /// <summary>
+    ///   Estimates the initial population numbers
+    /// </summary>
+    public static int CalculateMicrobePopulationInPatch(MicrobeSpecies species, Miche miche,
+        BiomeConditions biomeConditions, SimulationCache cache)
+    {
+        var energyBalanceInfo = cache.GetEnergyBalanceForSpecies(species, biomeConditions);
+        var individualCost = energyBalanceInfo.TotalConsumptionStationary + energyBalanceInfo.TotalMovement
+            * species.Behaviour.Activity / Constants.MAX_SPECIES_ACTIVITY;
```

Comment:
This math seems pretty complicated so extracting this as a method would be excellent to ensure there's just one source of truth for how much individual cost is in a species.